### PR TITLE
Cleanup classNames

### PR DESCRIPTION
**Pull request by tomheaton**
- remove redundant brackets
- use `clsx`
- matching quotes type

### CODE REVIEW

File: interface/app/$libraryId/TopBar/TopBarOptions.tsx [hunk 1]
```diff
@@ -117,9 +117,9 @@ export default ({ options }: TopBarChildrenProps) => {
 			</div>
 			<TopBarMobile
 				toolOptions={options}
-				className={
+				className={clsx(
 					windowSize <= 1279 && (toolsNotSmFlex?.length as number) > 0 ? 'flex' : 'hidden'
-				}
+				)}
```
Comment (utkubakir) [inline interface/app/$libraryId/TopBar/TopBarOptions.tsx:122]: clsx is not required here

File: interface/app/$libraryId/overview/index.tsx [hunk 2]
```diff
@@ -47,7 +48,9 @@ export const Component = () => {
 			<div className="flex flex-1">
 				<View
 					top={68}
-					className={settings.layoutMode === 'list' ? 'min-w-0' : undefined}
+					className={clsx(
+						settings.layoutMode === 'list' && 'min-w-0'
+					)}
```
Comment (utkubakir) [inline interface/app/$libraryId/overview/index.tsx:53]: clsx is not required here

File: interface/app/$libraryId/TopBar/TopBarOptions.tsx [hunk 3]
```diff
@@ -58,10 +58,10 @@ export default ({ options }: TopBarChildrenProps) => {
...
 								? 'both'
 								: index === 0
-								? 'left'
-								: index === group.length - 1
-								? 'right'
-								: 'none';
+									? 'left'
+									: index === group.length - 1
+										? 'right'
+										: 'none';
```
Comment (utkubakir) [inline interface/app/$libraryId/TopBar/TopBarOptions.tsx:64]: does prettier respect this? I feel like it will just format it back to how it is.

**Review by utkubakir — CHANGES_REQUESTED** [review]

**Comment by tomheaton** [comment]
changes made!

**Review by oscartbeaumont — APPROVED** [review]

**Review by utkubakir — APPROVED** [review]

**Comment by utkubakir** [comment]
Can we fix types?

**Comment by tomheaton** [comment]
where is this?

**Comment by utkubakir** [comment]
> where is this?

https://github.com/spacedriveapp/spacedrive/actions/runs/6186597174/job/16794620365?pr=1334
https://github.com/spacedriveapp/spacedrive/actions/runs/6186597174/job/16794619516?pr=1334

**Review by utkubakir — APPROVED** [review]

**Review by utkubakir — APPROVED** [review]

**Comment by utkubakir** [comment]
is this even worth it anymore 😆

**Comment by tomheaton** [comment]
Aha yeah, sorry I merged main using web and missed the duplicate imports.

**Pull request closed without merging**